monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+# **Monero: Un refugio a las severas leyes de decomiso civil (parte 1)** 
+*04/15/19*  
+Obiter Cryptum  
+
+
+El atropello excesivo, sistemático y atroz por parte del gobierno y de terceros sucede todos los días, y mientras que usted personalmente no lo experimenta, esta vivo y floreciendo. Los derechos fundamentales, los que se nos garantizan a nosotros los seres humanos a través de [La Declaración Universal de Derechos Humanos](https://www.un.org/es/universal-declaration-human-rights/) y las constituciones estatales, están siendo reducidos y usurpados, ya sea de manera evidente u oculta. 

```

Comment:
Next comments will be suggestion on words or a fraction of a phrase:

+ ... y mientras que usted personalmente no lo experimenta, _**sigue**_ vivo y floreciendo.

You can also use _**está**_, with just the _tilde_, both will be OK.

Reply:
I prefer the one you suggested, _**"sigue"**_, thanks @BlackLotus64 :) :+1:

Reply:
changed